test/e2e/utils.go

```diff
@@ -14,16 +14,13 @@ import (
 	"testing"
 	"time"
 
-	"github.com/jaegertracing/jaeger-operator/pkg/util"
-
 	osv1 "github.com/openshift/api/route/v1"
 	osv1sec "github.com/openshift/api/security/v1"
 	framework "github.com/operator-framework/operator-sdk/pkg/test"
 	"github.com/operator-framework/operator-sdk/pkg/test/e2eutil"
 	"github.com/sirupsen/logrus"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
-	"golang.org/x/net/context"
```

Comment:
This seems to be added via 5c37d2df44ee7f0017103bf2366b2caac897939e , but I guess this was supposed to be `context` instead. Just wanted to confirm with you if you really meant this old library, or if we can safely change it to use the stdlib's `context`.